api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/RPCEndpoint.java

```diff
@@ -12,47 +12,41 @@
 import edu.cornell.mannlib.vitro.webapp.dynapi.components.OperationResult;
 
 @WebServlet(name = "RPCEndpoint", urlPatterns = { "/api/rpc/*" })
-public class RPCEndpoint extends VitroHttpServlet  {
+public class RPCEndpoint extends VitroHttpServlet {
 
 	private static final long serialVersionUID = 1L;
- 	private static final Log log = LogFactory.getLog(RPCEndpoint.class);
+	private static final Log log = LogFactory.getLog(RPCEndpoint.class);
 	private ActionPool actionPool = ActionPool.getInstance();
-	
+
 	@Override
-	public void doGet( HttpServletRequest request, HttpServletResponse response ) {
-		process(request, response);
+	public void doGet(HttpServletRequest request, HttpServletResponse response) {
+		OperationResult.notImplemented().prepareResponse(response);
 	}
-	
+
 	@Override
-	public void doPost( HttpServletRequest request, HttpServletResponse response ) {
-		process(request, response);
+	public void doPost(HttpServletRequest request, HttpServletResponse response) {
+		String requestURL = request.getRequestURI();
```

Comment:
We should use request.getPathInfo() and verify it has at least one path. Else should return bad request.